fog/ledger/connection/src/key_image.rs

```diff
@@ -54,8 +55,9 @@ impl FogKeyImageGrpcClient {
                 .collect(),
         };
 
-        let response: CheckKeyImagesResponse =
-            self.conn.encrypted_enclave_request(&request, &[])?;
+        let response: CheckKeyImagesResponse = retry(Fixed::from_millis(100).take(5), || {
```

Comment:
This value, 100, is used in the other clients as well. Consider making it a shared constant with a descriptive name that's defined in the retry crate.

Reply:
okay, we can make the duration iterator a parameter and it can be in the sample paykit or something

Reply:
I'm going to fix this in follow up PR because i want to merge this, it will help with testing deployments and make fog test client work better in prod environments, and it's a lot of extra work if I do it right now, and that work is likely to conflict with other PRs that are inflight like yours I think.

I think this is important to do though and I have created a ticket to track it. Thanks for the suggestion